### ISSUE TITLE
Icon/border art

### ISSUE THREAD

**Issue by zachllogan** [random-person-001/MagicText#23]
First I want to ask if it is bad to open so may issues. I feel like it is good because then we know what needs to get worked on, but should issues only be opened when we know they are going to be worked on soon or should I have put all these art ones in a single issue?

I think the game should have a custom icon for the frame and eventually maybe a jar file. I would like to make this because I am in a webdesign class and we have been doing logos, so I would like to practice my design. Again mood board would be nice; I might make one.

As for the border art I am referring to the border (decoration) around the window, It is fine how it is, but do you want me to make a custom one? (be aware this might reduce performance slightly, so it might not be worth it, but I think it would be fine)

**Comment by HazilTheNut** [comment]
I like the simplicity of the border. It doesn't distract the player and it complements the rather plain appearance of SourceryText.

**Comment by random-person-001** [comment]
If you want to, you can make an icon for the jar file.  It wouldn't be high priority at all, but if you want to.

**Comment by zachllogan** [comment]
@HazilTheNut I agree (just thought I would ask because I enjoy doing that sort of thing). Though I would like to point out that a custom border wouldn't have to be complicated, it could be something similar to a DOS prompt, which seems rather fitting.

![](http://www.guidebookgallery.org/pics/gui/system/utilities/commandprompt/winnt40.png)

either way it probably doesn't need a custom border.

@random-person-001 Yeah, I was starting to try, but was having some problems with coming up with a design. The game is rather simple so the design should be as well and as the game is text based so should the design probably be. But I have very little space to work with (32x32) so I can't think of much besides a single '@' symbol, which seems fairly appropriate (and will do for now), but also seems a little unpersonal/ununique.

@HazilTheNut @random-person-001 do either of you have any ideas on how to make a more unique design that would fit in that small of a space (at most I might be able to get away with is about 8 text characters)? does the design have to be completely text-based?(probably?)
